app/src/main/java/io/neurolab/fragments/FocusVisualFragment.java

```diff
@@ -43,15 +43,14 @@
 import io.neurolab.utilities.PermissionUtils;
 
 import static android.app.Activity.RESULT_OK;
-import static io.neurolab.fragments.StatisticsFragment.convertToDouble;
 import static io.neurolab.utilities.FilePathUtil.LOG_FILE_KEY;
 
 public class FocusVisualFragment extends android.support.v4.app.Fragment {
 
     private static final int PERMISSION_REQUEST_WRITE_EXTERNAL_STORAGE_RESULT = 1;
     private boolean permission = false;
     private boolean isPlaying = false;
-    private String[] parsedData;
+    private static String[] parsedData;
```

Comment:
Local variable and this one both using the same name. Change to have a different name